pom.xml

```diff
@@ -268,7 +268,8 @@
           </execution>
         </executions>
         <configuration>
-          <scalaVersion>2.10.4</scalaVersion>
+          <scalaVersion>${scala.version}</scalaVersion>
+          <recompileMode>incremental</recompileMode>
```

Comment:
Interesting; so incremental compilation in Scala 2.11 either doesn't work correctly at all, or only works in SBT?

Maybe a better option would be to enable a different Maven profile in the presence of a jenkins environment variable?

That way we would get incremental compiles locally, which would be nice, unless it's so broken we don't think we want that.